pkgs/atc0005/send2teams/registry.yaml

```diff
@@ -7,281 +7,30 @@ packages:
-          algorithm: sha256
-        supported_envs:
-          - linux/amd64
-          - windows/amd64
-      - version_constraint: Version == "v0.9.2"
-        asset: send2teams-{{.Version}}-0-gdfae0b9-{{.OS}}-{{.Arch}}
-        format: raw
-        windows_arm_emulation: true
-        checksum:
-          type: github_release
-          asset: "{{.Asset}}.sha256"
-          algorithm: sha256
-        supported_envs:
-          - linux/amd64
-          - windows/amd64
-      - version_constraint: Version == "v0.9.3"
-        asset: send2teams-{{.Version}}-0-g948d8a3-{{.OS}}-{{.Arch}}
-        format: raw
-        windows_arm_emulation: true
-        checksum:
-          type: github_release
-          asset: "{{.Asset}}.sha256"
-          algorithm: sha256
-        supported_envs:
-          - linux/amd64
-          - windows/amd64
-      - version_constraint: Version == "v0.10.0"
-        asset: send2teams-{{.Version}}-0-g53606db-{{.OS}}-{{.Arch}}
-        format: raw
-        windows_arm_emulation: true
-        checksum:
-          type: github_release
-          asset: "{{.Asset}}.sha256"
-          algorithm: sha256
-        supported_envs:
-          - linux/amd64
-          - windows/amd64
-      - version_constraint: Version == "v0.10.1"
-        asset: send2teams-{{.Version}}-0-g5333670-{{.OS}}-{{.Arch}}
-        format: raw
-        windows_arm_emulation: true
-        checksum:
-          type: github_release
-          asset: "{{.Asset}}.sha256"
-          algorithm: sha256
-        supported_envs:
-          - linux/amd64
-          - windows/amd64
-      - version_constraint: Version == "v0.10.2"
-        asset: send2teams-{{.Version}}-0-g86b905e-{{.OS}}-{{.Arch}}
-        format: raw
-        windows_arm_emulation: true
-        checksum:
-          type: github_release
-          asset: "{{.Asset}}.sha256"
-          algorithm: sha256
-        supported_envs:
-          - linux/amd64
-          - windows/amd64
-      - version_constraint: Version == "v0.10.3"
-        asset: send2teams-{{.Version}}-0-g96a577a-{{.OS}}-{{.Arch}}
-        format: raw
-        windows_arm_emulation: true
-        checksum:
-          type: github_release
-          asset: "{{.Asset}}.sha256"
-          algorithm: sha256
-        supported_envs:
-          - linux/amd64
-          - windows/amd64
+      - version_constraint: semver("<= 0.10.3")
+        error_message: |
+          Versions 0.10.3 and earlier include a random string in the asset, making it difficult to predict the filename.
+          To avoid maintaining a registry with numerous overrides, we have decided not to support versions up to and including 0.10.3.
+          Please use a version later than 0.10.3.
       - version_constraint: "true"
         asset: send2teams-{{.OS}}-{{.Arch}}.{{.Format}}
         format: xz
-        windows_arm_emulation: true
+        files:
+          - name: send2teams
+            src: send2teams-{{.OS}}-{{.Arch}}
+        windows_arm_emulation: false
```

Comment:
Why do you disable `windows_arm_emulation`?

Reply:
I didn’t see any mention of ARM in the assets.
However, I did encounter issues due to a misconfiguration of the override for Windows. I hadn’t included the "name" field alongside "src" in the files, which caused problems with cmdx and on Windows containers. 
```
        files:
          - name: send2teams # without this param, windows test is broken
            src: send2teams-{{.OS}}-{{.Arch}}
```

This was part of my tests that I didn’t fix. I’m resubmitting it without those issues.

Reply:
Resolved in [`9bc661c`](https://github.com/aquaproj/aqua-registry/pull/29681/commits/9bc661c5bba9e2fb1e502e71e63436287ffd4751)